frontend/src/features.ts

```diff
@@ -1,16 +1,20 @@
 export interface Feature {
-  name: string;
+  name: FEATURE_KEY;
   description: string;
   active: boolean;
 }
 
-const features: Feature[] = [
-  {
-    name: 'v2',
-    description: 'Show v2 features',
-    active: false,
-  },
-];
+export enum FEATURE_KEY {
```

Comment:
enum is a type, so name it `FeatureKey`?

Reply:
That makes sense, I was thinking of using a const first, but later changed to enum. I forgot to update the name when doing it.